cmd/serve/ftp/ftp.go

```diff
@@ -60,6 +62,8 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.StringVarP(flagSet, &Opt.PassivePorts, "passive-port", "", Opt.PassivePorts, "Passive port range to use.")
 	flags.StringVarP(flagSet, &Opt.BasicUser, "user", "", Opt.BasicUser, "User name for authentication.")
 	flags.StringVarP(flagSet, &Opt.BasicPass, "pass", "", Opt.BasicPass, "Password for authentication. (empty value allow every password)")
+	flags.StringVarP(flagSet, &Opt.TLSCert, "client-cert", "", Opt.TLSCert, "TLS PEM key (concatenation of certificate and CA certificate)")
```

Comment:
I think these options should be called this to match the options available in `rclone serve http`

      --cert string                            SSL PEM key (concatenation of certificate and CA certificate)
      --key string                             SSL PEM Private key

If you copy the options from here then we can keep them consistent

https://github.com/rclone/rclone/blob/095c7bd801a8ef01bb4a9c792340fa3c0a302fe8/cmd/serve/httplib/httpflags/httpflags.go#L22-L23

Reply:
Sure

Reply:
@ncw I've made the requested changes.